mqemitter-redis.js

```diff
@@ -33,6 +34,14 @@ function MQEmitterRedis (opts) {
 
   var that = this
 
+  function onError (err) {
+    if (err) {
+      that.state.emit('emitError', err)
```

Comment:
why its this not `error`?

Reply:
@mcollina Because it I make this 'error' I get unhandled rejections from emit, check #15

Reply:
I think we need to track down why this is happening wqithout changing event name.

Reply:
@mcollina I have no clue Matteo... I have try to debug this but I can only tell you when this happens:

If emit throws an error and cb is not defined the unhandled rejection is thrown after I call `this.state.emit('error')`,  can that emit throw an error?

Reply:
Ok I have found this in docs: https://nodejs.org/api/events.html#events_class_events_eventemitter